app/models/user.rb

```diff
@@ -104,8 +104,8 @@ def student
 
   validates :name, :email, :country, :location, presence: true, unless: :github_import
 
-  accepts_nested_attributes_for :attendances, allow_destroy: true
-  accepts_nested_attributes_for :roles, allow_destroy: true
+  accepts_nested_attributes_for :attendances, allow_destroy: true, limit: 3
+  accepts_nested_attributes_for :roles
```

Comment:
@F3PiX that makes total sense! I think we should update the PR so that we don't remove it. I also try to figure out the impact of this PR. if we have users in the database with more than 3 attendances, rails would think that the users are invalid and users can't update their profiles (e.g.) anymore. since I don't have access to the database, I also posted that question in slack and hopefully somebody can help us out there.